language_service/src/completion.rs

```diff
@@ -343,8 +387,9 @@ impl CompletionListBuilder {
                                                     additional_edits.push((
                                                         protocol::Span { start, end: start },
                                                         format!(
-                                                            "open {};\n    ",
-                                                            namespace.name.clone()
+                                                            "open {};{}",
+                                                            namespace.name.clone(),
+                                                            indent_maker.get_ident(start),
```

Comment:
This gets called for every single completion we add to the list - almost every single global in the program. Should you cache the indent string?